oak/server/asylo/BUILD

```diff
@@ -170,8 +170,8 @@ enclave_loader(
 )
 
 enclave_loader(
-    name = "oak_app_loader",
-    srcs = ["asylo_oak_app_loader_main.cc"],
+    name = "oak_runner",
```

Comment:
What made you decide against `oak_runner_asylo` and `oak_runner_dev` in the end, which is what #569 references? Not necessarily against it either way, but I'm just curious why the change.

In any case, I think the binary name should match the source file name, or at least follow a similar structure.

Reply:
I just thought that the `asylo` is already implied by the directory.
Changed to `asylo_oak_runner`.
Setting `asylo_` at the beginning since all our files follow this convention.